linters/semgrep/plugin.yaml

```diff
@@ -0,0 +1,33 @@
+version: 0.1
+lint:
+  definitions:
+    - name: semgrep
+      files: [ALL]
+      runtime: python
+      package: semgrep
+      commands:
+        - name: check
+          output: sarif
+          # We use include filters instead of explicit paths because otherwise semgrep will run
+          # linters for every possible language for files that it does not recognize an extension for.
+          run: semgrep --config=auto --sarif --output=${tmpfile} --include=${target}
+          read_output_from: tmp_file
+          success_codes: [0]
+          batch: true
+          cache_results: true
+          # TODO(Chris): We need a new configuration option which means copy files into a sandbox.
```

Comment:
There's a couple todos like this one that remain. Should we make tickets to make sure they're tracked?

Reply:
I'll make an issue, thanks.